### PR TITLE
Feature -> Develop | [Flutter] Update basic section

### PR DESCRIPTION
**Pull request by we-prajapati-c001**
Task Video Proof: https://youtu.be/ScPV3pHHYnU
Updated video: https://youtu.be/zquAgQc1r1c

### CODE REVIEW

File: docs/vaahflutter/basics/getting-started.md [hunk 1]
```diff
@@ -9,22 +9,34 @@
 
 ## How to run the app
 
-### Step 1. Clone repository
+### Step 1. Clone Repository
```
Comment (themodernpk) [inline docs/vaahflutter/basics/getting-started.md:12]: @we-prajapati-c001 change `clone` to `download`

File: docs/vaahflutter/basics/getting-started.md [hunk 2]
```diff
@@ -9,22 +9,34 @@
...
 
 To clone the repository, you'll need to run the below command in your terminal. Make sure [git](https://git-scm.com/downloads) is installed in your system.
 
 ```bash
 git clone ...
```
Comment (themodernpk) [inline docs/vaahflutter/basics/getting-started.md:17]: @we-prajapati-c001 write this url to download the zip file `https://github.com/webreinvent/vaahflutter/archive/refs/heads/master.zip`

File: docs/vaahflutter/basics/getting-started.md [hunk 3]
```diff
@@ -9,22 +9,34 @@
 
 ## How to run the app
 
-### Step 1. Clone repository
+### Step 1. Clone Repository
 
 To clone the repository, you'll need to run the below command in your terminal. Make sure [git](https://git-scm.com/downloads) is installed in your system.
```
Comment (themodernpk) [inline docs/vaahflutter/basics/getting-started.md:14]: @we-prajapati-c001 no need to use the `git` remove it

File: docs/vaahflutter/basics/getting-started.md [hunk 4]
```diff
@@ -9,22 +9,34 @@
...
 ```bash
 git clone ...
 ```
 
-### Step 2. Run the app
+### Step 2. Basic Configuration
+
+There are two ways developers can integrate VaahFlutter in their project.
+
+1. Clone the repository and build your APP on top of it
```
Comment (themodernpk) [inline docs/vaahflutter/basics/getting-started.md:24]: @we-prajapati-c001 change `clone` to `download`

Comment (themodernpk) [reply docs/vaahflutter/basics/getting-started.md:24]: change `build` to `develop`

File: docs/vaahflutter/basics/getting-started.md [hunk 5]
```diff
@@ -9,22 +9,34 @@
...
+    - Developer needs to change some configurations for
+    - [bundle identifier and package name](update-basic-configs.md#_1-clone-the-repository-and-build-your-app-on-top-of-it)
+    - [app icon](update-basic-configs.md#_3-change-app-icon)
+    - and [splash screen](update-basic-configs.md#_4-change-splash-screen)
+
+2. Use VaahFlutter in your new/existing project
+    - Developer needs to copy few directories from cloned repository, check [this section](./update-basic-configs.md#_2-use-vaahflutter-in-your-new-existing-project).
```
Comment (themodernpk) [inline docs/vaahflutter/basics/getting-started.md:31]: @we-prajapati-c001 replace `clone` to download everywhere

**Review by themodernpk — CHANGES_REQUESTED** [review]

File: docs/vaahflutter/basics/getting-started.md (hunk 2 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/basics/getting-started.md:17]: yes sir

File: docs/vaahflutter/basics/getting-started.md (hunk 3 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/basics/getting-started.md:14]: yes sir

File: docs/vaahflutter/basics/getting-started.md (hunk 4 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/basics/getting-started.md:24]: yes sir

File: docs/vaahflutter/basics/getting-started.md (hunk 5 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/basics/getting-started.md:31]: yes sir

**Pull request merged**